web/src/features/fba/components/FBAMap.tsx

```diff
@@ -65,10 +45,30 @@ const FBAMap = (props: FBAMapProps) => {
   const [map, setMap] = useState<ol.Map | null>(null)
   const mapRef = useRef<HTMLDivElement | null>(null)
 
-  const fireCenterSource = buildFireVectorSource(fireCenterLayer)
+  const fireZoneVector = new VectorTileLayer({
+    // declutter: true,
```

Comment:
Stray comment

Reply:
I'd actually like to get the labels showing correctly, and the styling to match wf1 standards so it looks familiar - before merging this in anyway. Let's discuss with @tessmgood